documentation/src/main/asciidoc/mapper-pojo.asciidoc

```diff
@@ -69,6 +202,32 @@ The corresponding annotation is `org.hibernate.search.integrationtest.showcase.l
 * `org.hibernate.search.integrationtest.showcase.library.bridge.AccountBorrowalSummaryBridge` implements `TypeBridge`.
 The corresponding annotation is `org.hibernate.search.integrationtest.showcase.library.bridge.annotation.AccountBorrowalSummaryBridge`.
 
+=== Value bridges
+
+include::todo-placeholder.asciidoc[]
+
+[[mapper-pojo-bridge-valuebridge-builtin]]
+===== Built-in value bridges
+
+IMPORTANT: There are few supported types at the moment.
+This will be solved before the 6.0.0.Final release, see https://hibernate.atlassian.net/browse/HSEARCH-3047[HSEARCH-3047].
```

Comment:
Maybe a proper Asciidoc block would be nice?

Reply:
This is the short syntax. This is rendered as a proper Asciidoc admonition.